client/src/main/scala/drt/client/components/FlightsWithSplitsTable.scala

```diff
@@ -43,12 +41,15 @@ object FlightsWithSplitsTable {
                     originMapper: (String) => VdomNode = (portCode) => portCode,
                     splitsGraphComponent: SplitsGraphComponentFn = (_: SplitsGraph.Props) => <.div()
                    )(paxComponent: (Arrival, ApiSplits) => TagMod = (f, _) => f.ActPax) = ScalaComponent.builder[Props]("ArrivalsTable")
-
+//    .initialState(State(true))
     .renderPS((_$, props, state) => {
+
+
       val flightsWithSplits = props.flightsWithSplits
       val bestPax = props.bestPax
       val flightsWithCodeShares: Seq[(ApiFlightWithSplits, Set[Arrival])] = FlightTableComponents.uniqueArrivalsWithCodeShares(flightsWithSplits.flights)
-      val sortedFlights = flightsWithCodeShares.sortBy(_._1.apiFlight.PcpTime) //todo move this closer to the model
+      val sortedFlights = flightsWithCodeShares.sortBy(_._1.apiFlight.PcpTime)
+      //todo move this closer to the model
```

Comment:
this comment was at the end of the previous line. maybe it's irrelevant now anyway?